app/hero.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+
+from app.equipment.armor import Armor
```

Comment:
this creates a circular dependency because you import equipment here and then in equipment you import Hero. This is an antipattern. Refactor your code so your equipment doesn't have a dependency on Hero